test/05LessonFiveTest.js

```diff
@@ -4,7 +4,9 @@ var assert = require('assert');
 
 function loopIt() {
     // should return an array with numbers 0 - 99
```

Comment:
The requirement here states function should return an array. Think:
var array = [];
for(var i=0; i<100; i++){
//do something here to push the numbers to the array;
}

//return the array